python/openmmtorch.i

```diff
@@ -28,23 +29,40 @@
     }
 }
 
-%typemap(in) const torch::jit::Module&(torch::jit::Module module) {
+%typemap(in) const torch::jit::Module&(torch::jit::Module mod) {
     py::object o = py::reinterpret_borrow<py::object>($input);
-    module = torch::jit::as_module(o).value();
-    $1 = &module;
+    auto fileName = std::tmpnam(nullptr);
```

Comment:
There's probably a way of doing this without having to write it out to disk.  `torch::jit::load()` can take an `istream`, so if you can just write it out to a buffer in memory that will work.

Reply:
Turns out there was a way! hopefully it holds up for some versions :p